### ISSUE TITLE
0.1.4 is broken for property getter and setter

### ISSUE THREAD

**Issue by laggykiller** [cansik/nanobind-stubgen#12]
nanobind-stubgen 0.1.4 is broken for property getter and setter, it generates something like this:
```python
    @property
    def color_type(color_typescolor_typeecolor_typelcolor_typefcolor_type)color_type color_type-color_type>color_type color_typeicolor_typencolor_typetcolor_type:
        ...

    @color_type.setter
    def color_type(color_typescolor_typeecolor_typelcolor_typefcolor_type,color_type color_typeacolor_typercolor_typegcolor_type:color_type color_typeicolor_typencolor_typetcolor_type,color_type color_type/color_type)color_type color_type-color_type>color_type color_typeNcolor_typeocolor_typencolor_typeecolor_type:
        ...
```

In 0.1.3, it was correct:
```python
    @property
    def color_type(self) -> int:
        ...

    @color_type.setter
    def color_type(self, arg: int, /) -> None:
        ...
```

Step to reproduce using `apngasm_python`:
```
git clone https://github.com/laggykiller/apngasm-python.git
cd apngasm-python
pip install nanobind-stubgen==0.1.4
pip install .
nanobind-stubgen apngasm_python._apngasm_python
```

Note that this problem did not exist in 0.1.3

**Comment by cansik** [comment]
Yes, this is already discussed in #11

**Comment by laggykiller** [comment]
Seems to be fixed in 0.1.5
